example/gardener-local/kind/cluster/templates/_extra_port_mappings.tpl

```diff
@@ -50,6 +50,13 @@
 {{- end -}}
 {{- end -}}
 
+{{- define "extraPortMappings.gardener.operator.virtualGarden" -}}
+{{- if .Values.gardener.garden.deployed -}}
+- containerPort: 31443
+  hostPort: 443
```

Comment:
How does this relate to the settings above? Depending on the configuration setting port `443` could clash. #6997 added `listenAddress` to the above blocks so there should not be a problem, but it would mean that you could only access the virtual garden using your host's IP and not `localhost`/`127.0.0.1`.

Reply:
As discussed via Slack, currently it is not supported to use the same kind cluster for `gardener-operator` and the rest of Gardener at the same time. Hence, such clashes can only occur in theory. In the future, we might need to adapt in case want to support such scenario.